pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
which version of pnpm are you currently on?

Reply:
8.6.7